docker/docker-compose.yml

```diff
@@ -453,5 +453,22 @@ services:
       aws_cli:
         condition: service_healthy
 
+  camunda-tools:
+    healthcheck:
+      test: ["CMD", "java", "/app/scripts/HealthCheck.java", "http://localhost:8080/actuator/health", "||", "exit", "1"]
+      start_period: 1m
+      timeout: 30s
+      retries: 3
+    image: quay.io/ukhomeofficedigital/hocs-camunda-tools:0.0.1
+    ports:
+      - 8089:8080
+    networks:
+      - hocs-network
+    environment:
```

Comment:
More for the camunda app, but could you paramaterise the admin users id and password. We'll be changing that when we deploy to k8s, and reading from a secret I guess.
What I think happens is, the first time Cockpit runs, its adds the admin user to the workflow schema, and I think we have an opportunity to change the user and passord. Would be good to test the theory.